### PR TITLE
Feature/brpa 711 tiktok sdk

### PR DESCRIPTION
**Pull request by ConradoClark**
#### What is the purpose of this pull request?
- Create a `useTikTokSdk` react hook to load the required scripts.

#### What problem is this solving?
- Load the TikTok SDK when opening the admin page.

#### How to test it?
Open [this workspace](https://tiktok--sandboxbrdevbrl.myvtex.com/admin/tiktok) and then the Network tab in your browser. When the page is loading, it should load this script: `tbp_external_platform-v2.1`.

#### Screenshots or example usage
Screen Shot 2022-01-19 at 11 41 14

#### Types of changes

- [ ] Chore (non-breaking change which doesn't change any functionalities)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Review by anamaiabap — APPROVED** [review]

**Review by wagnerlduarte — APPROVED** [review]
LGTM

File: react/typings/tikTokProfile.d.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  bc: {
+    id: string
+    name: string
+  }
+  advId: string
+  advertiser: {
+    id: string
+    name: string
+  }
+  pixelCode?: string
```
Comment (felipefadul) [inline react/typings/tikTokProfile.d.ts:12]: Just to confirm: should it have `pixelCode?: string` on lines [12](https://github.com/vtex-apps/tiktok-tbp/pull/23/files#diff-96964d61a7402ae43b950c0b4fd22d5346cf1636514a45e19c6855ad293e0c72R12) and [14](https://github.com/vtex-apps/tiktok-tbp/pull/23/files#diff-96964d61a7402ae43b950c0b4fd22d5346cf1636514a45e19c6855ad293e0c72R14)?

**Review by felipefadul — APPROVED** [review]
Good! I only left a question to confirm.

File: react/typings/tikTokProfile.d.ts (hunk 1 above)
Comment (ConradoClark) [reply react/typings/tikTokProfile.d.ts:12]: Yes, I don't know why but some fields are way too redundant in this object 😅 but it's exactly like that in their example.

**Pull request merged**